patch_go1.14_test.go

```diff
@@ -0,0 +1,105 @@
+// Licensed under the MIT license, see LICENCE file for details.
+
+// +build go1.14
+
+package quicktest_test
+
+import (
+	"errors"
+	"os"
+	"testing"
+
+	qt "github.com/frankban/quicktest"
+)
+
+func TestPatchSetInt(t *testing.T) {
```

Comment:
Given that the implementation is such a thin wrapper around the underlying functionality, I think I'd just have a single smoke test just to make sure it's wired up ok rather than duplicating all the test logic.

Reply:
Done.